### ISSUE TITLE
ME network shortly going offline on auto save

### ISSUE THREAD

**Issue by Programie** [AppliedEnergistics/Applied-Energistics-2#4060]
I recently added a Quantum Network Bridge to my ME network to connect a second location to my ME network. Since connecting the QNB I had really weird issues: Every ~40 seconds the whole ME network on the side of the ME Controller goes down for ~1 second (the controller is powered off for 1 second), then switches to status "Missing channel" and after 2 or 3 seconds switches back to online. But it only happens on the side which is connected to the Controller! The other side of the QNB stays online all the time.

I already searched in the server logs for any hints and also enabled the chunkloader logging which gave me some glue: Every time the issue appears, the chunks are unloaded and reloaded. Obviously, the whole ME network is offline while the chunks are unloaded.

Once I disconnect the QNB, everything works fine. It also only happens on the network which is connected to the QNB. A separate network without any QNB doesn't have any issues at all even if the Controller and QNB which have issues are located in the same chunk.

I also tried a minimal ME setup only containing a power source (EnderIO Capacitor Bank), the ME Controller, a ME Terminal and one part of the QNB on one side and a ME Terminal as well as the second part of the QNB on the other side in a completely different region on the map which didn't had those issues.

I guess the chunks are always unloaded but in the completely different region it happens much faster than in the region where my main base including the ME network is located.

I already read about the player and/or server admin has to ensure that the chunks containing the ME network are always loaded. But if that is really the issue, why is the remote site (connected to the ME Controller via the QNB) always online and never losing the connection but the main site containing the ME Controller is losing it?

Video showing the issues: https://www.youtube.com/watch?v=jLHg3UKWMmE

- Minecraft Version: 1.12.2
- AE2 Version: rv6.stable build 6
- Forge Version: 14.23.5.2837

**Comment by mindforger** [comment]
Isnt quite a bit of chunks around the spawn area loaded permanently?

**Comment by Programie** [comment]
You mean trying to move the world spawn to the chunk with the QNB and ME Controller and testing it again?

But it still does not explain why there are no issues at all if the QNB is not connected to the network.

**Comment by mindforger** [comment]
There are plenty sources that can cause chunks to „flicker“ and when the QNB unloads or reloads, the me network has tot update. Install chunkloader on both ends and in case of chickenchunks loader put a spotloader on top of the chunkloader or you may encounter unloaded chunkloaders on startup or loading a game

**Comment by Programie** [comment]
I just gave Chicken Chunks a try and I the result: The issues are gone!

I placed a Chunk Loader Block next to the QNB on the remote site, returned to my base and checked the ME terminals which previously gone offline every ~40 seconds. And now, even after minutes, ME is still online (keeping the terminal opened while writing this and it's still open).

So it's just caused by chunks not staying active? But it's also weird why the chunks are reloaded again if they are "not active"? Shouldn't they stay unloaded if they are not in use? And why do I not have any issues if I test a minimal ME setup in completely different regions on the same map?

Anyway... it doesn't look like an issue with AE2 so I'm going to close this issue. Thanks for your help!
